cypress/integration/proposal_administration.ts

```diff
@@ -137,4 +137,61 @@ context('Proposal administration tests', () => {
 
     cy.get('[placeholder="Search"]').should('have.value', 'test');
   });
+
+  it('Should be able to save table sort state in url', () => {
+    let officerProposalsTableAsTextBeforeSort = '';
+    let officerProposalsTableAsTextAfterSort = '';
+
+    cy.login('user');
+    cy.createProposal(
+      'Aaaaaaaaa test proposal title',
```

Comment:
I see no reason for multiple 'Aaaaaaaaa'. A typo?

Reply:
I have added multiple `Aaaaaaa` just because here I am testing the sorting by title if it is kept in the url prams after reloading. There is already one proposal with a random title and this one I wanted to be always on top if the sort order is ascending and always at the bottom if the sort order is descending. Maybe it looks a bit weird but didn't have any better idea how to do it 😅